simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/component/exception/AvailabilityStateVariableAccessor.java

```diff
@@ -49,12 +52,21 @@ public String getVariableName() {
     }
 
     private GetVariableResult getActualValue(Component component, Variable variable, GetVariableDatum.AttributeType attributeType) {
-        return new GetVariableResult()
+        Integer evseId = component.getEvse().getId();
+        Optional<Evse> optionalEvse = getStation().getState().tryFindEvse(evseId);
+
+        GetVariableResult getVariableResult = new GetVariableResult()
                 .withComponent(component)
                 .withVariable(variable)
-                .withAttributeType(GetVariableResult.AttributeType.fromValue(attributeType.value()))
-                .withAttributeValue(new CiString.CiString1000("AVAILABLE"))
-                .withAttributeStatus(GetVariableResult.AttributeStatus.ACCEPTED);
+                .withAttributeType(GetVariableResult.AttributeType.fromValue(attributeType.value()));
+
+        if (optionalEvse.isPresent()) {
```

Comment:
Looks like there is a code duplication. Can you extract this logic to a shared method?

Reply:
will do here: https://github.com/evbox/station-simulator/pull/11